### PR TITLE
#148574727 Write tests for all actions, reducers and components. 

### PR DESCRIPTION
**Pull request by gbengaPS**
#### What does this PR do?
-add and remove codes based on new understanding
#### Description of Task to be completed?
-refactor reducers
-add jsdoc to functions
-refactor backend test to different files
-move API doc folder
-modify UI fonts
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:

### CODE REVIEW

File: server/includes/functions.js [hunk 1]
```diff
@@ -65,12 +63,32 @@ export const checkInputLength = (requestObject, inputField) => {
...
 };
+
+/**
+ *
+ *
+ * @param {string} password
+ * @returns {string} -returns hashed password
+ */
+export const encryptPassword = (password) => {
+  let returnValue;
```
Comment (johnkingzy) [inline server/includes/functions.js]: I would suggest you try to use variable names that are explicit enough. `returnValue` could be `encryptedPassword`

File: test/message.test.js [hunk 2]
```diff
@@ -26,6 +26,22 @@ describe('Message tests', () => {
...
+      messageBody: 'this is a message',
+      messagePriority: 'something'
+    };
+    supertest(app).post('/api/group/1/message')
+    .set('x-access-token', token)
+    .send(message)
+    .end((err, res) => {
+      assert.equal(res.statusCode, 400);
+      assert.equal(res.body.error,
+        'Validation error: Message priority can only Normal, Urgent, or Critical');
```
Comment (johnkingzy) [inline test/message.test.js]: I think there's a typo in this line. `Validation error: Message priority can only Normal, Urgent, or Critical` could be `Validation error: Message priority can only be Normal, Urgent, or Critical`

File: server/includes/functions.js (hunk 1 above)
Comment (gbengaPS) [reply server/includes/functions.js]: Nice. Thanks for that

File: test/message.test.js (hunk 2 above)
Comment (gbengaPS) [reply test/message.test.js]: Thanks. Changing it immediately.

**Pull request merged**